visualization/dashboard/src/MLIDashboard/Controls/AxisConfigurationDialog/AxisConfigDialog.styles.ts

```diff
@@ -0,0 +1,108 @@
+import { FontSizes, FontWeights, getTheme, IProcessedStyleSet, IStyle, mergeStyleSets, ICalloutContentStyles } from "office-ui-fabric-react";
+
+export interface IAxisControlDialogStyles {
+    axisConfigDialog:IStyle;
+    wrapper:IStyle;
+    leftHalf:IStyle;
+    rightHalf:IStyle;
+    detailedList:IStyle;
+    spinButton:IStyle;
+    selectButton: IStyle;
+    featureText:IStyle;
+    filterHeader:IStyle;
+    featureComboBox:IStyle;
+    treatCategorical:IStyle;
+}
+
+export const axisControlDialogStyles: () => IProcessedStyleSet<IAxisControlDialogStyles> = () => {
+    const theme = getTheme();
+    return mergeStyleSets<IAxisControlDialogStyles>({
+        axisConfigDialog: {
+            position: 'absolute',
+            overflowY: 'visible',
+            width: '560px',
+            height: '386px',
+            left: '450px',
+            top: '250px',
+            //TODO: fix rgba here
+            boxShadow: '0px 0.6px 1.8px rgba(0, 0, 0, 0.108), 0px 3.2px 7.2px rgba(0, 0, 0, 0.132)',
+            borderRadius: '2px'
+        },
+        wrapper: {
+            height: "238px",
+            width: "560px",
+            display: "flex",
+            marginTop:"65px"
+        },
+        leftHalf: {
+            width: "213px",
+            height: "238px",
+            marginLeft: "40px",
+        },
+        rightHalf: {
+            display: "inline-flex",
+            width: "255px",
+            height: "238px,",
+            flexDirection: "column",
+            background: theme.palette.neutralLight,
+            marginRight: "27px",
+            marginLeft: "25px",
+            //marginTop: "0px",
+            borderRadius: "5px"
+        },
+        filterHeader: {
+            fontWeight: FontWeights.semibold,
+            fontSize: FontSizes.medium,
+            color: theme.palette.black
+        },
+        detailedList: {
+            marginTop: "28px",
+            height: "160px",
+            width: "197px",
+            overflowX: "visible"
```

Comment:
should these kind of repeating strings be constants?

Reply:
these aren't really repeating strings in the normal sense, these are what whould normally be css. The IStyle interface enumerates what strings are acceptable for each key.